Packs/SimpleSFTP/Integrations/SimpleSFTP/SimpleSFTP.py

```diff
@@ -33,15 +39,15 @@ def main():
                 'Type': entryTypes['note'],
                 'ContentsFormat': formats['text'],
                 'Contents': res,
-                'ReadableContentsFormat': formats['text'],
-                'HumanReadable': tableToMarkdown('The directory files:', res),
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable': tableToMarkdown('', res, ["Directory Files"]),
```

Comment:
Why did you remove the title? Please restore.

Reply:
Added title for MD